### PR TITLE
Login Page

### PR DESCRIPTION
**Pull request by rahul0396**
https://user-images.githubusercontent.com/43301962/154839487-08e1bad1-e0a6-4c3b-adb3-3dcfee0cb137.mp4

### CODE REVIEW

File: css/login.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+body{
+    background-color: #F7F7F7;
+}
+.login-box{
+    position: absolute;
+    /* width: 500px;
+    height: 350px; */
+    left: 30%;
+    top: 20%;
+    /* background-color: white; */
```
Comment (MaruthiPothuganti) [inline css/login.css:12]: Commented code can be removed as it is unnecessary

File: pages/login.html [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+<link rel="stylesheet" href="/css/login.css">
+<body>
+   
+   <!-- navbar start -->
+   <div class="navbar-section">
+    <nav class="navbar p-0p5 flex m-0">
+        <div class="brand-name m-0p5 fs-1p5">Naturewell</div>
+
+        <div class="search-bar my-0p5 mx-auto">
+            <input class="search-input fs-1p2 px-1 py-0 border-none" type="text" placeholder="Search for products">
```
Comment (MaruthiPothuganti) [inline pages/login.html:22]: input type of search can be used for search bar

**Review by MaruthiPothuganti — COMMENTED** [review]
Suggested a couple of changes. Everything else looks good

**Review by Ronak-Mutha — COMMENTED** [review]
Remove extra comments and you can use variables for colors and spacing. Rest all good to merge.
Note: These are just suggestions, try to work on them after submission.

**Pull request merged**